dogecoin/params.go

```diff
@@ -141,9 +127,9 @@ var MainNetParams = chaincfg.Params{
 	Bech32HRPSegwit: "bc", // always bc for main net
 
 	// Address encoding magics
-	PubKeyHashAddrID:        0x00, // starts with 1
-	ScriptHashAddrID:        0x05, // starts with 3
-	PrivateKeyID:            0x80, // starts with 5 (uncompressed) or K (compressed)
+	PubKeyHashAddrID:        0x1E, // 30, starts with 1
+	ScriptHashAddrID:        0x16, // 22, starts with 3
+	PrivateKeyID:            0x9E, // 158, starts with 5 (uncompressed) or K (compressed)
 	WitnessPubKeyHashAddrID: 0x06, // starts with p2
 	WitnessScriptHashAddrID: 0x0A, // starts with 7Xh
```

Comment:
Address encoding:
- Need to update comments, not sure what the correct values are other than D for the pubkey one?
- Not sure what the values should be for Witness

Reply:
- starts with D
- starts with 9 or A
- starts with Q